doc/views_api.md

```diff
@@ -0,0 +1,100 @@
+
+## `StateComponent` API
+
+<pre>
+&lt;class&gt; StateComponent extends Backbone.BaseView
```

Comment:
Actually `StateComponent` seems to be an abstract class for all `StepViews`, what do you think ?